src/webots/vrml/WbProtoTemplateEngine.cpp

```diff
@@ -82,11 +82,6 @@ bool WbProtoTemplateEngine::generate(const QString &logHeaderName, const QVector
   tags["context"] +=
     QString("webots_version: {major: '%1', revision: '%2'}").arg(version.toString(false)).arg(version.revisionNumber());
 
-  if (templateLanguage == "lua") {
-    tags["fields"] = convertStatementFromJavaScriptToLua(tags["fields"]);
-    tags["context"] = convertStatementFromJavaScriptToLua(tags["context"]);
-  }
-
   return WbTemplateEngine::generate(tags, logHeaderName, templateLanguage);
```

Comment:
I believe we could get rid of this `templateLanguage` parameter.

Reply:
Will do. I'd thought you might want to keep it in case you decided to support multiple languages again in the future.